.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Question
+about: Ask any question about this project or ask for help with using it
+title: ''
+labels: ''
```

Comment:
We've got a question label. Let's enforce that: https://github.com/nhsuk/ui-test-core/labels/question